charts/rasa-x/templates/db-migration-service.yaml

```diff
@@ -0,0 +1,21 @@
+{{ if eq "true" (include "db-migration-service.requiredVersion" .) }}
+{{ if .Values.separateDBMigrationService }}
```

Comment:
optional: slightly inconsistent to the other uses of `and` instead of two `if`s for this condition